integration/log_rotation_test.go

```diff
@@ -111,6 +120,13 @@ func (s *LogRotationSuite) TestTraefikLogRotation(c *check.C) {
 	c.Assert(lineCount, checker.GreaterOrEqualThan, 7)
 }
 
+func logAccessLogFile(c *check.C, fileName string) {
+	output, err := ioutil.ReadFile(fileName)
```

Comment:
Maybe a case of [`t.Helper()`](https://golang.org/pkg/testing/#T.Helper)?

Reply:
In the case of the integration tests we can not use it, as we don't have access to `testing.T`.

Reply:
Ah -- sad and true.